containers/Home/About/About.tsx

```diff
@@ -1,14 +1,19 @@
 import { StyledAbout, StyledLabel, StyledContent } from './styles'
 
 import type { Props } from './interface'
+import moment from 'moment'
 
 export const About: React.FC<Props> = ({ about }) => {
     return (
         <StyledAbout>
             {about.map((item, index) => (
                 <div key={index}>
-                    <StyledLabel>{item.label}</StyledLabel>{' '}
-                    <StyledContent>{item.content}</StyledContent>
+                    <StyledLabel>{item.label}</StyledLabel>
+                    <StyledContent>
+                        {typeof item.content === 'number'
+                            ? Number(moment().format('YYYY')) - Number(moment(item.content).format('YYYY'))
+                            : item.content}
```

Comment:
Please separate this into a new variable. Beside, I don't think `typeof item.content === 'number'` is a good solution in the future, but we will refactor this later so this should be good to go